### PR TITLE
Man  misc fix

### PR DESCRIPTION
**Pull request by masatake**
Small fixes.

### CODE REVIEW

File: man/ctags.1.rst.in [hunk 1]
```diff
@@ -757,6 +764,10 @@ built-in language names.
 	used in the ``--language-force``, ``--languages``, ``--<LANG>-kinds``,
 	and ``--regex-<LANG>`` options.
 
+``--maxdepth``
+	limits the depth of directory recursion enabled with the ``--recurse``
```
Comment (k-takata) [inline man/ctags.1.rst.in]: limits -> Limits

File: man/ctags.1.rst.in [hunk 2]
```diff
@@ -769,8 +780,11 @@ built-in language names.
 	the automatic reading of any configuration options from either a file
 	or the environment (see FILES).
 
+``--print-language``
+	Just prints the parsers for specified input files, and then exits.
+
 ``--quiet[=yes|no]``
-		Write fewer messages(default is no).
+	Write fewer messages(default is no).
```
Comment (k-takata) [inline man/ctags.1.rst.in]: Maybe it's better to insert a space before `(`.

**Comment by masatake** [comment]
Updated. Thank you.

**Comment by k-takata** [comment]
LGTM.

**Pull request merged**